code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -206,7 +206,9 @@
 		hud = new()
 
 	//If the species has eyes, they are the default vision organ
-	if(!vision_organ && has_organ["eyes"])
+	if(name == "Diona")
```

Comment:
This is an unfortunate hack to begin with. _Why_ did people think this was smart _aaa._

Remove this. Just set `vision_organ = "head"` in the definition of the diona datum.

Reply:
Sorry, I did not see for a sec diona datum